OWNERS

```diff
@@ -1,7 +0,0 @@
-approvers:
```

Comment:
Are we ok to remove this @rhatdan ? I don't know all the differences between these owners files.